### DISCUSSION TITLE
There are no vehicles in spawn vehicle

### DISCUSSION THREAD

**Discussion by Romangelion** [YimMenu/YimMenu#1853] [Q&A]
![252147682-b994a5d7-eb79-4ad0-bd43-e59bcdc6285d](https://github.com/YimMenu/YimMenu/assets/140587795/921803f3-e70d-4af5-8e72-f53bf0880a0d)
I've looked up  if someone had reported this before and they did, but in the end they didn't upload the cout log file they had been requested to in order to help them,  so I'll leave that too:
[cout.log](https://github.com/YimMenu/YimMenu/files/12168880/cout.log)

**Comment by Sergus01** [comment]
It happened to me too with the STEAM version of the game. I changed the version using that directly from the desktop and everything works.
